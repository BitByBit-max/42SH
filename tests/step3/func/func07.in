var=elise

main()
{
    echo "hello $var"
    if true; then
        echo "how are you?";
    fi
    echo $1
} > in

main good
cat in
rm in
