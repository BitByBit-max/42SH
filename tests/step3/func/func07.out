hello elise
how are you?
good
